tests/basic/sharedv4_svc_test.go

```diff
@@ -749,6 +695,157 @@ var _ = Describe("{Shared4SvcUnexportExport}", func() {
 	})
 })
 
+// Stop PX on the client node and wait for the server to remove the export for client.
+// Then, do a failover and restart PX on the client node.
+var _ = Describe("{Shared4SvcClientOfflineTooLong}", func() {
+	var testrailID = 54778
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/54778
+	var runID int
+	var contexts, testSv4Contexts []*scheduler.Context
+	var workers []node.Node
+
+	JustBeforeEach(func() {
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	It("has to stop PX on the client node long enough for server to remove the export then bring the node back", func() {
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("client-offline-%d", i))...)
+		}
+		testSv4Contexts = getTestSv4Contexts(contexts)
+		if len(testSv4Contexts) == 0 {
+			Skip("No test-sv4-svc apps were found")
+		}
+		workers = node.GetWorkerNodes()
+
+		Step("scale the test-sv4-svc apps so that one pod runs on each worker node", func() {
+			scaleApps(testSv4Contexts, len(workers))
```

Comment:
quick question, this is assuming k8s scales evenly to the worker nodes. There won't be a case where one node runs two pod. Is this assumption correct?
If not, is there a way we can ensure each node runs a pod?

Reply:
The Deployment spec of the application used in these tests have a podAntiAffinity which ensures that only one pod runs on a node.

Reply:
ah got it. thanks! :)